recipes/get_recipes.go

```diff
@@ -32,52 +51,58 @@ func MakeRequest(c *gin.Context, nutrient string, cookTime string) {
 	json.Unmarshal(responseData, &responseObject)
 
 	var data []map[string]any
+	var instructions []string
 
 	for _, recipe := range responseObject.Recipes {
+		instructions = scraper.ScrapeInstructions(recipe.RecipeInfo.Url)
+
+		if instructions == nil {
```

Comment:
I like that we have this option. Especially since the case is we have fewer recipes that this happens so it's nice to have this to fall back.